src/graphforge/executor/evaluator.py

```
@@ -226,8 +228,46 @@ def _evaluate_string_function(func_name: str, _args: list[CypherValue]) -> Cyphe
         ValueError: If function is unknown
         TypeError: If arguments have invalid types
     """
-    # Placeholder - will be implemented in Feature 2
-    raise ValueError(f"String function not yet implemented: {func_name}")
+    if func_name == "LENGTH":
+        # LENGTH(string) -> int
+        if not isinstance(args[0], CypherString):
+            raise TypeError(f"LENGTH expects string, got {type(args[0]).__name__}")
```

Ruff (0.14.14)

[warning] Avoid specifying long messages outside the exception class

(TRY003)